pkg/kubelet/config/file_linux.go

```diff
@@ -58,9 +58,11 @@ func (s *sourceFile) startWatch(logger klog.Logger) {
 		}
 
 		if err := s.doWatch(logger); err != nil {
-			logger.Error(err, "Unable to read config path", "path", s.path)
 			if _, retryable := err.(*retryableError); !retryable {
```

Comment:
`path does not exist` is the only usage of `retryableError` at this time. I don't think "retryable" is the right concept for this, because a "retry" occurs in all cases (the backoff is what differs). I chose to keep this change tiny instead of reworking that.